docs/elements/examples/sign-in.mdx

```diff
@@ -28,7 +28,6 @@ Before you build your sign-in flow, you need to configure the appropriate settin
 ```tsx {{ filename: '/app/sign-in/[[...sign-in]]/page.tsx', collapsible: true }}
 'use client';
 
-// If you are getting type errors, please set 'moduleResolution' to 'bundler' in your tsconfig
```

Comment:
This should not be removed. See https://clerkinc.slack.com/archives/C05ST2R2HM3/p1715010356592819?thread_ts=1714984418.298789&cid=C05ST2R2HM3 and https://clerkinc.slack.com/archives/C05ST2R2HM3/p1714660364172489?thread_ts=1714625535.748349&cid=C05ST2R2HM3 and https://clerkinc.slack.com/archives/C05ST2R2HM3/p1715004287363359 for more context from me.